scripts/build/docker/Dockerfile.template

```diff
@@ -16,6 +16,8 @@ RUN apt-get update \
     curl \
     fuse \
     git \
+    graphicsmagick \
```

Comment:
I suggest to extend electron-builder Docker image. `electronuserland/electron-builder:latest` https://github.com/electron-userland/electron-builder/wiki/Docker